### ISSUE TITLE
Add .zip decompression format support to Promtail

### ISSUE THREAD

**Issue by 4bdulla** [grafana/loki#10728]
**Is your feature request related to a problem? Please describe.**
I would like to use `.zip` decompression format for processing log files using Promtail as an agent. However only `gzip`, `bzip `an `z` formats are supported.

**Describe the solution you'd like**
Being able to use `zip` format in decompression mechanism of promtail's configuration:

```yaml
decompression:
  enabled: true
  format: zip
```

**Describe alternatives you've considered**
Unfortunately in order to use different decompression mechanism, I'm supposed to update compression mechanism in all systems. 

**Additional context**
None

**Comment by paladium** [comment]
I agree, having that feature would immensely help process logs that are rotated and compressed when loki is down

**Comment by MahammadAgayev** [comment]
That's a useful one to have!

**Comment by cstyan** [comment]
Hello, thanks for your PR.

We're currently reevaluating promtails position as a project within Grafana Labs. Internally we're actually using the [Agent](https://github.com/grafana/agent) for both metrics and logs collection at this point. Additionally, the agent team is more likely to have time to dedicate to your PR.

While we haven't made a formal decision yet, we expect in the near future that all new feature work will be done in the Agent's log collection pipelines rather than in Promtail. As such, we're not accepting new features in Promtail right now but only bug and security fixes.

**Comment by JStickler** [comment]
Promtail is end of life (EOL) as of March 2, 2026. No future support or updates will be provided. All future feature development will occur in [Grafana Alloy](https://github.com/grafana/alloy).
